saleor/graphql/webhook/mutations.py

```diff
@@ -200,3 +204,29 @@ def perform_mutation(cls, _root, info, **data):
                 )
 
         return super().perform_mutation(_root, info, **data)
+
+
+class EventDeliveryRetry(BaseMutation):
+    delivery = graphene.Field(EventDelivery, description=".")
+
+    class Arguments:
+        id = graphene.ID(
+            required=True, description="ID of the event delivery to retry."
+        )
+
+    class Meta:
+        description = "Retries event delivery."
+        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        error_type_class = WebhookError
+        error_type_field = "webhook_errors"
```

Comment:
This field adds deprecated errors. We don't need deprecated error to a new mutation.  
```suggestion
```

Reply:
fixed